diff --git a/node_modules/react-native-inset-shadow/src/index.js b/node_modules/react-native-inset-shadow/src/index.js
index 76dbbc3..d237417 100644
--- a/node_modules/react-native-inset-shadow/src/index.js
+++ b/node_modules/react-native-inset-shadow/src/index.js
@@ -4,34 +4,12 @@ import PropTypes from 'prop-types'
 import styles from './styles'
 import Shadows from './shadows'
 
-const InsetShadow = ({ 
-  children,
-  containerStyle,
-  left,
-  top,
-  right,
-  bottom,
-  shadowColor,
-  shadowRadius,
-  shadowOffset,
-  shadowOpacity,
-  elevation
-}) => {
-  const shadowProps = {
-    left,
-    top,
-    right,
-    bottom,
-    shadowColor,
-    shadowRadius,
-    shadowOffset,
-    shadowOpacity,
-    elevation
-  }
+const InsetShadow = (props) => {
+
   return (
-    <View style={[styles.container, containerStyle]}>
-      {children}
-      <Shadows {...shadowProps} />
+    <View style={[styles.container, props.containerStyle]}>
+      {props.children}
+      <Shadows {...defaultShadowProps} {...props} />
     </View>
   )
 }
@@ -60,8 +38,7 @@ InsetShadow.propTypes = {
   elevation: PropTypes.number,
 }
 
-InsetShadow.defaultProps = {
-  containerStyle: {},
+const defaultShadowProps = {
   left: true,
   top: true,
   right: true,
diff --git a/node_modules/react-native-inset-shadow/src/shadows.js b/node_modules/react-native-inset-shadow/src/shadows.js
index 098d975..4623bbf 100644
--- a/node_modules/react-native-inset-shadow/src/shadows.js
+++ b/node_modules/react-native-inset-shadow/src/shadows.js
@@ -5,9 +5,10 @@ const shadowTypes = ['left', 'top', 'right', 'bottom']
 
 const Shadows = (props) => {
   return shadowTypes.map((shadow, i) => {
-    if(!props[shadow]) {
+    if(props[shadow] != null && !props[shadow]) {
       return null
     }
+
     const { shadowColor, shadowOpacity, shadowOffset, shadowRadius, elevation } = props
     const shadowProps = {
       shadowColor,
@@ -22,12 +23,4 @@ const Shadows = (props) => {
   })
 }
 
-// default to show all edges
-Shadows.defaultProps = {
-  left: true,
-  top: true,
-  right: true,
-  bottom: true
-}
-
 export default Shadows
\ No newline at end of file
